debian/control

```diff
@@ -17,16 +17,13 @@ Build-Depends: debhelper (>= 9),
                libxml2-utils,
                maliit-framework-dev (>= 0.99.0+git20130615+97e8335-0ubuntu3),
                pkg-config,
-               python3:any,
-               python3-flake8 (>= 2.2.2-1ubuntu4) | python3-flake8:native,
```

Comment:
Keep these three lines, but change this one to be just `python3-flake8`. You'll be surprised :-)